backend/api/userService.js

```diff
@@ -1,12 +1,54 @@
 const crypto = require("crypto");
 const userModel = require("../models/user");
+const commentModel = require("../models/comment");
+const postModel = require("../models/post");
 const roles = require("../models/role");
-const { notBannedUser, userNotExists } = require("../middlewares/validation");
```

Comment:
behöver vi inte kolla om user är bannad också? Har bannad user lov att ta bort sitt konto?

Reply:
man ska vara inloggad för att kunna ta bort sitt konto. man kan inte logga in om man är bannad... darför behövs det inte